Podcast/BookmarkTableViewCell.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  BookmarksTableViewCell.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 3/20/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+protocol BookmarkTableViewCellDelegate: class {
+    
+    func bookmarkTableViewCellDidPressPlayPauseButton(bookmarksTableViewCell: BookmarkTableViewCell)
+    func bookmarkTableViewCellDidPressRecommendButton(bookmarksTableViewCell: BookmarkTableViewCell)
+    func bookmarkTableViewCellDidPressMoreActionsButton(bookmarksTableViewCell: BookmarkTableViewCell)
+}
+
+class BookmarkTableViewCell: UITableViewCell {
+
+    ///
+    /// Mark: View Constants
+    ///
+    static let height: CGFloat = 96
+    let height: CGFloat = BookmarkTableViewCell.height
+    let separatorHeight: CGFloat = 1
+    let padding: CGFloat = 18
+    let episodeImageSideLength: CGFloat = 60
+    let episodeNameLabelY: CGFloat = 18
+    let episodeNameLabelX: CGFloat = 90
+    let episodeNameLabelHeight: CGFloat = 21
+    let dateTimeLabelX: CGFloat = 90
+    let dateTimeLabelY: CGFloat = 41.5
+    let dateTimeLabelHeight: CGFloat = 14.5
+    
+    let buttonTitlePadding: CGFloat = 7
+    
+    let playButtonX: CGFloat = 90
+    let playButtonBottomY: CGFloat = 18
+    let playButtonHeight: CGFloat = 15
+    let playButtonWidth: CGFloat = 75
+    
+    let recommendedButtonX: CGFloat = 148
+    let recommendedButtonXPlaying: CGFloat = 168
+    let recommendedButtonBottomY: CGFloat = 18
+    let recommendedButtonHeight: CGFloat = 15
+    let recommendedButtonWidth: CGFloat = 60
+    
+    let moreButtonRightX: CGFloat = 18
+    let moreButtonBottomY: CGFloat = 18
+    let moreButtonHeight: CGFloat = 15
+    let moreButtonWidth: CGFloat = 15
+    
+    ///
+    /// Mark: Variables
+    ///
+    var heightConstraint: NSLayoutConstraint!
+    var episodeImage: UIImageView!
+    var episodeNameLabel: UILabel!
+    var dateTimeLabel: UILabel!
+    var recommendedButton: UIButton!
+    var moreButton: UIButton!
+    var playButton: UIButton!
+    var separator: UIView!
+    
+    weak var delegate: BookmarkTableViewCellDelegate?
+    
+    
+    ///
+    ///Mark: Init
+    ///
+    
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        frame.size.height = height
```

Comment:
why are you setting the frame's height here? it's just going to be set later by the tableview